quiche/src/recovery/cubic.rs

```diff
@@ -226,21 +227,8 @@ fn on_packet_acked(r: &mut Recovery, packet: &Acked, now: Instant) {
     // When the recovery episode ends with recovering
     // a few packets (less than cwnd / mss * ROLLBACK_THRESHOLD_PERCENT(%)), it's
     // considered as spurious and restore to the previous state.
-    if r.congestion_recovery_start_time.is_some() {
-        let new_lost = r.lost_count - r.cubic_state.prior.lost_count;
-        let rollback_threshold = (r.congestion_window / r.max_datagram_size) *
-            ROLLBACK_THRESHOLD_PERCENT /
-            100;
-        let rollback_threshold = rollback_threshold.max(MIN_ROLLBACK_THRESHOLD);
-
-        if new_lost < rollback_threshold {
-            let did_rollback = rollback(r);
-
-            if did_rollback {
-                return;
-            }
-        }
-    }
+    //
+    // Spurios loss detection is handled in [`Recovery`]
```

Comment:
If I recall correctly the prior behavior was added to address cubic over-reacting to spurious loss.

It's confusing the have the comment above that mentions ROLLBACK_THRESHOLD_PERCENT when this commit removes that constant entirely.

We need to discuss if this refactor commit should be changing the functionality of cubic like this.

Reply:
i brought it back